rasa/core/tracker_store.py

```diff
@@ -917,33 +917,44 @@ def get_db_url(
         )
 
     def _create_database_and_update_engine(self, db: Text, engine_url: "URL"):
-        """Create databse `db` and update engine to reflect the updated `engine_url`."""
-
+        """Creates database `db` and updates engine accordingly."""
         from sqlalchemy import create_engine
 
+        if not self.engine.dialect.name == "postgresql":
+            rasa.shared.utils.io.raise_warning(
+                f"The parameter 'login_db' can only be used with a postgres database.",
```

Comment:
Seems a bit a wrong place to warn about `login_db`, since it was passed as `login_db` in the constructor of `SQLTrackerStore`.

Reply:
It's called from the constructor, no? And we can only safely determine the dialect once we're connected 🤷🏻